### DISCUSSION TITLE
Got approved by student but cannot use copilot

### DISCUSSION THREAD

**Discussion by rooberpk16** [community/community#142530] [GitHub Education] [answered]
### Hi everyone,

Hello, i had some trouble with github student to approve me finally (although i've been using it for some years) but i finally got it approved last week. The issue i'm having right now is that, when i log in with my github account in VSCode, i get [auth] Invalid copilot token: missing token: 403. What can i do?

**Comment by davevad93** [comment, marked as answer]
Hi @rooberpk16 ,

Since you have the `PRO` badge I assume your application status has a purple label and says `Congrats! Your request for @rooberpk16 was approved and your benefits are now available.`  which means you should have full access to your student benefits. First of all, make sure you have redeemed your free `Copilot` subscription by going to the [Pack offers page](https://education.github.com/pack), scroll down to the “***All Offers***” section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), redeem your offer and then check if you can use it, if you still can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Reply by rooberpk16** [reply]
Thanks!

**Reply by JohnDelatto** [reply]
Tell the team to kindly please restore my account status

**Reply by JohnDelatto** [reply]
> Hi @rooberpk16 ,
> 
> Since you have the `PRO` badge I assume your application status has a purple label and says `Congrats! Your request for @rooberpk16 was approved and your benefits are now available.` which means you should have full access to your student benefits. First of all, make sure you have redeemed your free `Copilot` subscription by going to the [Pack offers page](https://education.github.com/pack), scroll down to the “_**All Offers**_” section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), redeem your offer and then check if you can use it, if you still can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

Tell the team to kindly please restore my account status 

Thanks for the support
